src/components/tables/RowActions/Delete/Progress.tsx

```diff
@@ -0,0 +1,90 @@
+import { Box, CircularProgress, ListItemText, Stack } from '@mui/material';
+import { createEntityDraft } from 'api/drafts';
+import { createDraftSpec } from 'api/draftSpecs';
+import { createPublication } from 'api/publications';
+import Error from 'components/shared/Error';
+import { useEffect, useState } from 'react';
+import { FormattedMessage } from 'react-intl';
+
+interface Props {
+    deleting: any;
+    onFinish: (response: any) => void;
+}
+
+enum States {
+    DELETING = 1,
+    FAILED = 2,
+    DELETED = 3,
+}
+
+function DeleteProgress({ deleting, onFinish }: Props) {
+    const [state, setState] = useState<States>(States.DELETING);
+    const [error, setError] = useState<any | null>(null);
+
+    useEffect(() => {
+        const failed = (response: any) => {
+            console.log('response.error', response.error);
+
+            setState(States.FAILED);
+            setError(response.error);
+            onFinish(response);
+        };
+
+        const succeeded = (response: any) => {
+            setState(States.DELETED);
+            onFinish(response);
+        };
+
+        const makeDeleteCall = async (spec: any) => {
```

Comment:
Would either `requestEntityDeletion` or `requestEntityRemoval` work as function name alternatives?

Reply:
I don't see any meaningful difference in those namings and this one.